GameServerCore/Enums/DamageSource.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameServerCore.Enums
+{
+    public enum DamageSource
```

Comment:
This doesn't match packets definition of DamageSource:
```
    public enum DamageSource : uint
    {
        Raw = 0x0,
        InternalRaw = 0x1,
        Periodic = 0x2,
        Proc = 0x3,
        Reactive = 0x4,
        OnDeath = 0x5,
        Spell = 0x6,
        Attack = 0x7,
        Default = 0x8,
        SpellAoe = 0x9,
        SpellPersist = 0xA,
        Pet = 0xB,
    }
```
If you just moved this from somewhere else then you can safely ignore this comment and we can solve this in another PR.

Reply:
Moved it from lib to the API.